app/views/payments/paymentHistory.scala.html

```diff
@@ -45,13 +45,22 @@ <h2>@model.selectedYear.toString</h2>
 
       @if(model.transactions.isEmpty || model.transactions.forall(_.clearedDate.isEmpty)) {
         <p>@messages("paymentsHistory.noHistory", model.selectedYear.toString)</p>
+
       } else {
         <table>
           <thead>
             <tr>
-              <th scope="col">@messages("paymentsHistory.paymentReceivedTitle")</th>
-              <th scope="col">@messages("paymentsHistory.descriptionTitle")</th>
-              <th class="numeric" scope="col">@messages("paymentsHistory.amountTitle")</th>
+              <th scope="col"> 
```

Comment:
Could you update the table heading tests in some way because of this change? Perhaps rewording the test descriptions and updating the CSS selectors to use `visuallyhidden` just for sanity that they are in fact being hidden?